nupic/research/frameworks/vernon/parser_utils.py

```diff
@@ -89,7 +89,7 @@
 RAY_PARSER.add_argument("-s", "--with-server", action="store_true",
                         help="Start Ray Tune API server")
 RAY_PARSER.add_argument("--single_instance", action="store_true",
-                        help="Uses single instance run method")
+                        help="Uses single instance run method", default=False)
```

Comment:
You dont need to add `default=False` when using `action="store_true"`.
See https://docs.python.org/3.7/library/argparse.html?highlight=store_true#action